.gitmodules

```diff
@@ -8,7 +8,7 @@
 	branch = master
 [submodule "third-party/ViGEmClient"]
 	path = third-party/ViGEmClient
```

Comment:
The only thing that could possibly change would be the submodule name and path.

Maybe we should do that just to avoid anything with the trademarked vigem name?

Reply:
I don't know if it's worth it. "ViGEm" appears all over the place in that submoduke, like in function names, comments, and header file names.

Reply:
Yea, true. I don't have any issue with merging this.